### PR TITLE
React: Introduce Qr code (yarn.lock updated)

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

**Comment by xitronix** [comment]
Screenshot 2019-10-25 at 19 43 02
Screenshot 2019-10-25 at 19 43 24

**Comment by xitronix** [comment]
Screenshot 2019-10-25 at 19 37 49
Screenshot 2019-10-25 at 19 39 01

**Comment by Marik-D** [comment]
@alexcherman Please review

File: universal-login-react/src/ui/styles/topUpDefaults.sass [hunk 1]
```diff
@@ -36,6 +36,14 @@
...
       text-transform: uppercase
 
+    .qr-code-wrapper
+      text-align: center
+      border-radius: 6px
+      border: none
+      background: #362F6A
+      margin: 15px 0px
+      padding: 15px 0px
+
```
Comment (alexcherman) [inline universal-login-react/src/ui/styles/topUpDefaults.sass]: Remove layout styles(margin, padding) from here. In topUpDefaults should be only styles associated with appearance.

Comment (xitronix) [reply universal-login-react/src/ui/styles/topUpDefaults.sass]: ✔️

**Comment by xitronix** [comment]
image
image

**Pull request merged**